examples/demo_input_paste.py

```diff
@@ -7,5 +7,8 @@ def paste():
             print(repr(e))
             time.sleep(1)
 
+            if e == u'<ESC>':
```

Comment:
So that there isn't a 1 second delay first, could you move this to the top of the for loop body?